app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -110,6 +113,18 @@ open class FenixApplication : Application() {
         registerActivityLifecycleCallbacks(visibilityLifecycleCallback)
 
         components.core.sessionManager.register(NotificationSessionObserver(this))
+
+        if (!DateUtils.isToday(this.settings.lastPlacesStorageMaintenance)) {
+            runStorageMaintenance()
+        }
+    }
+
+    private fun runStorageMaintenance() {
+        GlobalScope.launch {
+            bookmarkStorage()?.runMaintenance()
```

Comment:
`bookmarks` and `history` are both handled by the same `writer` instance internally - they both sit on top of the same sqlite db file. Underneath, both of these calls will end up as the same call to Rust: `places_run_maintenance`.
So, just pick one - doesn't matter which - and add a comment to explain why it's just one and not both.

Once we add logins, it'll need its own maintenance call, since it's backed by a different db file.